test-app/src/main/java/org/readium/r2/testapp/drm/DrmManagementFragment.kt

```diff
@@ -122,20 +118,17 @@ class DrmManagementFragment : Fragment() {
                         .onSuccess {
                             val result = DrmManagementContract.createResult(hasReturned = true)
                             setFragmentResult(DrmManagementContract.REQUEST_KEY, result)
-                        }.onFailure { exception ->
-                            exception.report(requireView())
                         }
+                        .onFailure { handle(it) }
                 }
             }
             .show()
     }
+
+    private fun handle(error: DrmManagementViewModel.DrmError) {
+        error.toUserError().show(requireActivity())
+    }
 }
 
 private fun Date?.toFormattedString() =
     DateTime(this).toString(DateTimeFormat.shortDateTime()).orEmpty()
-
-// FIXME: the toast is drawn behind the navigation bar
-private fun DrmManagementViewModel.DrmError.report(view: View) {
-    Snackbar.make(view, toUserError().getUserMessage(view.context), Snackbar.LENGTH_LONG).show()
-    Timber.w(error.toDebugDescription())
-}
```

Comment:
Is this fixed?

Reply:
Most likely, it's using the Activity's content view instead of the fragment. I didn't test it out though.